orderapi/src/main/java/fi/hel/verkkokauppa/order/service/order/OrderService.java

```diff
@@ -119,7 +123,13 @@ public String generateOrderId(String namespace, String user, String timestamp) {
     public Order createByParams(String namespace, String user) {
         String createdAt = DateTimeUtil.getDateTime();
         String orderId = generateOrderId(namespace, user, createdAt);
-        Order order = new Order(orderId, namespace, user, createdAt);
+        Long incrementId;
+        try {
+            incrementId = this.incrementId.generateOrderIncrementId();
```

Comment:
Should that generateOrderIncrementId function handle the exception and return null or generated id?

Reply:
Okay I changed it. Generally, I'd consider this bad practice since it hides errors and we can no longer trust that something is actually the value we want it to be without null checks here and there. However, this seems to be the approach used in this project so far.